staging/src/k8s.io/client-go/util/workqueue/queue.go

```diff
@@ -47,16 +49,33 @@ type QueueConfig struct {
 }
 
 // New constructs a new work queue (see the package comment).
+//
+// Deprecated: use NewTyped instead.
 func New() *Type {
 	return NewWithConfig(QueueConfig{
 		Name: "",
 	})
 }
 
+// NewTyped constructs a new work queue (see the package comment).
+func NewTyped[T comparable]() *TypedType[T] {
```

Comment:
`TypedType` is stuttering and looks kind of funny. Maybe just `Typed`?

Reply:
Changed it